detection/meson.build

```diff
@@ -183,29 +183,17 @@ endif
 fft_small_opt = get_option('fft_small')
 
 fft_small_arm_code = '''
-#include <gmp.h>
-#if GMP_LIMB_BITS != 64
-# error
-error
-#endif
-
 #include <arm_neon.h>
 #if !(defined(__GNUC__) && defined(__ARM_NEON))
 # if !(defined(_MSC_VER) && defined(_M_ARM64))
 #  error
 error
 # endif
 #endif
-void main(){};
+int main(){return 0;}
```

Comment:
Don't know if this will be relevant or not in the future, but not writing `void` in the parameter list may induce an error when compiling with C23.

Reply:
I don't know enough about this. Is it expected that you write `int main(void)` these days?

Reply:
Hmm, it seems like I was in the wrong. We got some warnings related to not putting anything into the parameter list, but I cannot remember why that was.